webapp/javascript/pages/IntroPages/NotFound/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import cx from 'classnames';
+import { Link } from 'react-router-dom';
+import styles from '../IntroPages.module.css';
+
+function NotFoundPage() {
```

Comment:
nice touch

Reply:
Now it's handled with go server, but here is a switch so make it work on react side